scripts/upower/95-upower-hid.rules

```diff
@@ -23,6 +23,7 @@ ENV{DEVTYPE}=="usb_interface", GOTO="up_hid_end"
 ATTRS{idVendor}=="03f0", ENV{UPOWER_VENDOR}="Hewlett Packard"
 ATTRS{idVendor}=="0463", ENV{UPOWER_VENDOR}="Eaton"
 ATTRS{idVendor}=="047c", ENV{UPOWER_VENDOR}="Dell"
+ATTRS{idVendor}=="0483", ENV{UPOWER_VENDOR}=""
```

Comment:
@clepple @aquette : Should there be a non-trivial string here?

Reply:
The proper name for 0483 is STMicro. There is another `nutdrv_qx` model (TS Shara) which also uses 0483.

The problem is that `nut-usbinfo.pl` is probably going to rewrite this string anyway. At some point, we will need to rewrite `usbhid-ups` to not make so many assumptions about USB VIDs.

Reply:
TODO/Note to self: the macro in C file that defines vendor ID should be preceded by a comment with vendor name. Then the script picks it up.

Reply:
Updated in source, will do same in existing master branch (TS Shara) to be in sync string-wise